CMakeLists.txt

```diff
@@ -116,6 +116,7 @@ option(VELOX_ENABLE_PARQUET "Enable Parquet support" OFF)
 option(VELOX_ENABLE_ARROW "Enable Arrow support" OFF)
 option(VELOX_ENABLE_REMOTE_FUNCTIONS "Enable remote function support" OFF)
 option(VELOX_ENABLE_CCACHE "Use ccache if installed." ON)
+option(VELOX_ENABLE_SDK "Enable SDK support." ON)
```

Comment:
It's not immediately clear what we mean by "SDK" here. Maybe "VELOX_ENABLE_JNI_BINDINGS" could be more explicit?

Reply:
we could use `velox/jni` as the base dir